src/compat/browser_compatibility_types.ts

```diff
@@ -252,6 +254,7 @@ export interface IMediaElement extends IEventTarget<IMediaElementEventMap> {
   srcObject?: undefined | null | MediaProvider;
   textTracks: TextTrackList | never[];
   volume: number;
+  disableRemotePlayback: boolean;
```

Comment:
Isn't it basically also optional here?

Reply:
you mean it's optional because it's not implemented on every browser ? firefox has no attribute `disableRemotePlayback`

Reply:
I have made it optional in the IMediaElement.

There were no error because `HTMLMediaElement` has the property disable remote playback defined as a non-optional boolean